changelogs/current.yaml

```diff
@@ -45,6 +45,10 @@ removed_config_or_runtime:
 - area: runtime
   change: |
     Removed runtime guard ``envoy_reloadable_features_boolean_to_string_fix`` and legacy code paths.
+- area: runtime
```

Comment:
nit: it is probably not "runtime", but "sni".

Reply:
oops. fixed :)